### ISSUE TITLE
False positive: 'color-contrast' rule fails for disabled control's label

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#1731]
Per WCAG 1.4.3 the text that is part of an inactive user interface component do not have contrast requirement.

> Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Expectation: 'color-contrast' would not fail for labels related to disabled controls

Actual: 'color-contrast' rule fails for a label(using ```for``` attribute) of a disabled control

Motivation: Text related to the disabled control is exempt from the contrast requirements

Demo link: https://developer.microsoft.com/en-us/fabric#/controls/web/combobox

![image](https://user-images.githubusercontent.com/22068434/62076429-7be52300-b1fc-11e9-935b-c459ac6e24d0.png)

<pre><code>axe-core version: 3.3.1-canary.ed28298
axe-Coconut version: 3.8.1</code></pre>

**Comment by straker** [comment]
Thanks for reporting the issue. We'll take a look into what's causing the issue.

**Comment by chandana7393** [comment]
Tested, working fine on Axe-coconut. Able to reproduce the issue in Axe and Attest.
![disabled_color_contrast](https://user-images.githubusercontent.com/45285854/64001705-0b486380-cb26-11e9-95bf-a3237904a8b0.PNG)
![disabled_color_contrast_attest](https://user-images.githubusercontent.com/45285854/64001713-0c799080-cb26-11e9-8180-2f9113b86c13.PNG)
**Tested Environment:**
Attest - 2.6.1.24122v
Axe - 3.9.0.24122v
Axe-coconut - 3.9.0.24122v
Chrome - 76.0.3809.132v
Firefox - 68.0.2v
OS - Windows 10 64 bit.
